openIt/jsonld-contexts/openIt-compound.jsonld

```diff
@@ -0,0 +1,10 @@
+{
+  "@context": [
+    "https://easy-global-market.github.io/ngsild-api-data-models/user/user.jsonld",
+    "https://easy-global-market.github.io/ngsild-api-data-models/device/jsonld-contexts/device.jsonld",
+    "https://easy-global-market.github.io/ngsild-api-data-models/openIt/jsonld-contexts/gate.jsonld",
+    "https://easy-global-market.github.io/ngsild-api-data-models/openIt/jsonld-contexts/key.jsonld",
+    "https://easy-global-market.github.io/ngsild-api-data-models/openIt/jsonld-contexts/authorization.jsonld",
```

Comment:
You can also add the Authorization context for Stellio rights it will be useful for sure: `authorization/jsonld-contexts/authorization.jsonld`